content/articles/domain-auto-renewal.md

```diff
@@ -14,34 +14,33 @@ categories:
 
 ---
 <warning>
-Auto-renewals begin the renewal process 30 days before the domain is set to expire. You cannot turn off the auto-renewal process during this time. We deliver auto-renewal and expiration notices starting 2 months prior to the domain's expiration date. Please update your auto-renewal preferences before it reaches the 30-day threshold.
+Auto-renewals begin 30 days before a domain is set to expire. The auto-renewal process cannot be stopped or prevented during this 30-day period. We deliver auto-renewal and expiration notices starting two months before a domain's expiration date. Please update your preferences before the 30-day threshold.
 </warning>
 
 Auto-renewal reduces the risk of a domain expiring by renewing the domain automatically within 30 days of expiration. When auto-renewal is turned on for a domain, we'll attempt to renew it automatically prior to expiration. We'll bill your credit card if the renewal is successful.
 
```

Comment:
What happens if it's _unsuccessful_?

Reply:
@itsalyse Typically, if a domain auto-renewal fails, it’s because we couldn’t collect the payment on time. When this happens, we disable auto-renewal, so the customer will need to renew their domain manually. We also send an email notifying the customer that the payment couldn’t be processed.

this is the message we send to them:

<p>The payment for the auto-renewal fee for <%= @domain.to_display %> failed. The payment gateway failed with: <%= @payment_error.code %> (<%= @payment_error.decline_code %>).</p> <p>As a result of this failure we have proceeded to disable auto-renewal on your domain, and it won't be automatically renewed.</p> <h3>What's Next</h3> <ol> <li>Update your payment details to make sure you can pay for the renewal of your domain.</li> <li>Manually renew your domain if possible.</li> <li>Enable domain auto-renewal back on your domain. Make sure all of your domains have auto-renewal enabled so you never lose a domain due to expiration.</li> <li><%= link_to "Contact us", feedback_url %> if you need any assistance, we're happy to help.</li> </ol>


Maybe it might be helpful to add the reasons a domain auto-renewal can fail, which are:
Payment couldn't be collected: In this case, the customer will need to renew the domain manually.
Customer unsubscribed: If they’ve unsubscribed from their account, auto-renewal is automatically disabled.